services/admin.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ * parses then return all parent ids
+ * @param {Number} unitId numeric admin unit id
```

Comment:
This function looks like it expects a string, and should probably do a string conversion just to be safe anyway.